index.html

```diff
@@ -1,14 +1,113 @@
 <!DOCTYPE html>
 <html>
 <head>
-<link rel="stylesheet" type="text/css" href="style.css">
+ <link rel="stylesheet" type="text/css" href="style.css">
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+   <meta charset="utf-8">
```

Comment:
I see some duplication here, please make sure you include just one type of bootstrap, jquery ... right now you have a jquery version of: 3.3.1 and 3.3.7, same for bootstrap. When you include/import something, you want to let the browser know that you are using a library. So you need to include it once. In our case you need an import for a jquery, for bootstrap ( css and js ), and that's all :)